eng/devices/android.cake

```diff
@@ -132,18 +135,34 @@ Task("Build")
 	.WithCriteria(!string.IsNullOrEmpty(PROJECT.FullPath))
 	.Does(() =>
 {
-	MSBuild(PROJECT.FullPath, c => {
-		c.Configuration = CONFIGURATION;
-		c.Restore = true;
-		c.Properties["ContinuousIntegrationBuild"] = new List<string> { "false" };
-		c.Targets.Clear();
-		c.Targets.Add("Build");
-		c.Targets.Add("SignAndroidPackage");
-		c.BinaryLogger = new MSBuildBinaryLogSettings {
-			Enabled = true,
-			FileName = BINLOG,
-		};
-	});
+	if (USE_DOTNET)
+	{
+		DotNetCoreBuild(PROJECT.FullPath, new DotNetCoreBuildSettings {
```

Comment:
how does this know what version of dotnet to use?

Should it be using the stuff we have inside dotnet.cake to setup a local install of dotnet and use that?

Reply:
This uses the dotnet that is installed here: https://github.com/dotnet/maui/pull/1099/files#diff-b69b78f4a2fcf116ee0e1d0bfc8ffa31a05469eb6a3585e486af367aeddf498bR44